mk/python.mk

```diff
@@ -0,0 +1,12 @@
+PYTHON = python3
+PYTHON := $(shell which $(PYTHON))
+ifndef PYTHON
```

Comment:
Add extra condition if `python` indicated by `$PATH` is exactly of version 3. Think of the case somebody built Python3 from source and the generated executable is `python`.